phpcs.xml.dist

```diff
@@ -9,7 +9,7 @@
     <!-- Ignore warnings, show progress of the run and show sniff names -->
     <arg value="nps"/>
 
-    <config name="php_version" value="80300"/>
+    <config name="php_version" value="80400"/>
```

Comment:
Why is this line even needed? This is no library.

Reply:
True. I removed it.